node-graph/gcore/src/raster.rs

```diff
@@ -53,6 +53,32 @@ impl<N: Node<(), Output = f32> + Copy> BrightenColorNode<N> {
 	}
 }
 
+#[derive(Debug, Clone, Copy)]
+pub struct GammaColorNode<N: Node<(), Output = f32>>(N);
+
+impl<N: Node<(), Output = f32>> Node<Color> for GammaColorNode<N> {
+	type Output = Color;
+	fn eval(self, color: Color) -> Color {
+		let gamma = self.0.eval(());
+		let per_channel = |col: f32| col.powf(gamma);
```

Comment:
The current GammaNode for an image actually does the inverse of this (gamma correction?) - which approach doe we want @Keavon?

Reply:
We flow linear colors through our node graph, it's up to any particular node to gamma or inverse gamma the color if necessary (like when importing or exporting an image). Besides that, I don't have enough context or understanding to comment about this part of the code. Happy to discuss it though if it's important.

Reply:
The gamma node that already exists in the node graph actually does gamma correction. Do we want to change this?

Reply:
It's a useful node to have, so there's no need to change it at the moment.

Reply:
We should at some point transition the node that works on images to use a per channel node internally.
We could also have the node system use the correct implementation based on the input type (Image or Color)